containers/build_own_image-README.md

```diff
@@ -0,0 +1,6 @@
+# Building your own Kraken dashboard image
+- Git clone the Kraken repository using git clone https://github.com/krkn-chaos/krkn-dashboard.git
+- Execute ` podman build -t <new_image_name>:latest . ` in the containers directory within krkn-dashboard to build an image from a Dockerfile.
```

Comment:
The command needs to be: $ podman build -t <new_image_name>:latest -f containers/dockerfile to be able to copy the source code into the container.